SDWebImage/SDAnimatedImageView.h

```diff
@@ -61,7 +61,7 @@
  You can specify a runloop mode to let it rendering.
  Default is NSRunLoopCommonModes on multi-core iOS device, NSDefaultRunLoopMode on single-core iOS device
  */
-@property (nonatomic, copy, nonnull) NSRunLoopMode runLoopMode NS_AVAILABLE_IOS(3_1);
+@property (nonatomic, copy, nonnull) NSRunLoopMode runLoopMode;
```

Comment:
This API does no effect on macOS platform. So I mark this as `NS_AVAILABLE_IOS` to indicate this will be only available on iOS/tvOS.

If you think this not so obvious, we can replace it with `API_UNAVAILABLE(macos)`